docs/strategies/retry.md

```diff
@@ -108,6 +108,55 @@ new ResiliencePipelineBuilder<HttpResponseMessage>().AddRetry(optionsExtractDela
 | `OnRetry`          | `null`                                                                     | Action executed when retry occurs.                                                       |
 | `MaxDelay`         | `null`                                                                     | Caps the calculated retry delay to a specified maximum duration.                         |
 
+## Calculation of the next delay
+
+If the `ShouldHandle` predicate returns `true` and the next attempt number is not greater than `MaxRetryAttempts` then the retry strategy calculates the next delay.
+
+There are many properties that are somehow contribute to this calculation:
+
+- `Delay`
+- `DelayGenerator`
+- `MaxDelay`
+- `UseJitter`
+- `BackoffType`
+
+> [!IMPORTANT]
+> The below described algorithm is an implementation detail. It might change in the future without further notice.
+>
+> Also please bear in mind that some details are not mentioned here to keep the algorithm description short and terse.
+
+The `BackoffType` property's data type is a [`DelayBackOffType`](xref:Polly.DelayBackOffType) enum. This controls primarily how the calculation is done.
+
+### Constant
+
+One might think that here we only put into the account the `Delay` property. In reality all above listed properties are used.
```

Comment:
I'm not a fan of the style of wording here - it reads weird here to be making assertions about what the reader might think, rather than just explaining it.

Reply:
```suggestion
Even though the `Constant` name could imply that only the `Delay` property is used, in reality all the above listed properties are used.
```

Is it better?